docs/getting-started/setup/macos.md

```diff
@@ -267,6 +267,9 @@ You should be using Yarn 1.22.19 or later.
 
 # Install Podman
 
+> If you are using Docker Desktop, you can skip this section, and set an environment variable via `export USE_DOCKER_DESKTOP=1` in your shell profile.
+> This will enable Ryuk for cleaning up old containers under testing.
+
```

Comment:
This seems to affect all environments, so should we add something to the `windowsos.md` as well?

Reply:
Good catch! Pushed https://github.com/objectcomputing/check-ins/pull/2440/commits/7102c02f938a1bc5d3d8f4d26b8d9eda2106875b